src/pages/cli/start/install.mdx

```diff
@@ -69,3 +69,7 @@ Successfully set up the new user.
 ### Work within your frontend project
 
 After you install the CLI, navigate to a JavaScript, iOS, or Android project root, initialize AWS Amplify in the new directory by running `amplify init`. After a few configuration questions, you can use amplify help at any time to see the overall command structure. When you’re ready to add a feature, run `amplify add <category>`.
+
+## Configuring the CLI when there is an existing IAM user
```

Comment:
I'd remove this H2 and instead move the content below right above of "Work within your frontend project". as a

> Note: ...